FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2ScaffoldActivity.kt

```diff
@@ -59,8 +59,6 @@ class V2ScaffoldActivity : V2DemoActivity() {
     }
 
     override val paramsUrl = "https://github.com/microsoft/fluentui-android/wiki/Controls#params-25"
-    override val controlTokensUrl =
```

Comment:
Lets have the url as 25, so that other component control token url number matches with paramsurl. Lets keep 25 for this and make it empty

Reply:
There is no ControlToken for Scaffold. Hence, removed.  The no. assigned in link in gitub page. To have same no. for control taken & params, then scaffold need to put it at end. Then the page wont be arranged alphabetically.

Reply:
Will add ControlToken heading for Scaffold with description no control token available.